README.md

```diff
@@ -290,7 +290,7 @@ Outgoing http calls to Redis made using StackExchange.Redis library can be autom
                 .SetSampler(Samplers.AlwaysSample)
                 .UseZipkin()
                 .SetResource(new Resource(new Dictionary<string, string>() { { "service.name", "my-service" } }))
-                .AddCollector(t =>
+                .AddAdapter(t =>
                 {
                     var collector = new StackExchangeRedisCallsCollector(t);

```

Comment:
this one `StackExchangeRedisCallsCollector` should also need to change

Reply:
There are more 'collectors' in readme, would you mind changing it too?

Reply:
Sure I forgot all about the readme. We also need to remember too change the package links once the packages have been released.